### PR TITLE
Minor editorial nits based on comment by Justin U

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-ice-pac.xml [hunk 1]
```diff
@@ -129,7 +129,7 @@
         </t>
         <section title="No Candidates From Peer">
           <t>
-            Per RFC XXXX, an ICE agent could provide zero candidates of
+            Per RFC XXXX, an ICE agent might provide zero candidates of
```
Comment (juberti) [inline draft-ietf-ice-pac.xml]: I still think 'may' is the ideal word here. 'can' could also work.

File: draft-ietf-ice-pac.xml [hunk 2]
```diff
@@ -216,14 +216,12 @@
...
-        ICE session to Failed as long as the timer is running. As a result, the ICE agent will
-        not remove any data streams or set the state of the ICE session to Failed as long as
-        the timer is running.
+        ICE session to Failed as long as the timer is running.
       </t>
       <t>
         When the timer eventually elapses, the ICE agent MUST resume typical
-        ICE processing, including setting the state of a checklist to Failed if there
-        are no pairs left to check, and handling any consequences as indicated
+        ICE processing, including setting the state of a checklist to Failed if they
```
Comment (juberti) [inline draft-ietf-ice-pac.xml]:
```suggestion
        ICE processing, including setting the state of any checklists to Failed if they
```

File: draft-ietf-ice-pac.xml [hunk 3]
```diff
@@ -216,14 +216,12 @@
...
+        hace no pairs left to check, and handling any consequences as indicated
```
Comment (juberti) [inline draft-ietf-ice-pac.xml]:
```suggestion
        have no pairs left to check, and handling any consequences as indicated
```

**Review by juberti — CHANGES_REQUESTED** [review]

File: draft-ietf-ice-pac.xml (hunk 1 above)
Comment (cdh4u) [reply draft-ietf-ice-pac.xml]: Ok, I will use "can".

File: draft-ietf-ice-pac.xml (hunk 2 above)
Comment (cdh4u) [reply draft-ietf-ice-pac.xml]: Will fix as suggested.

File: draft-ietf-ice-pac.xml (hunk 3 above)
Comment (cdh4u) [reply draft-ietf-ice-pac.xml]: Will fix.

**Review by juberti — APPROVED** [review]

**Pull request merged**